packages/server/src/analytics/logic.test.ts

```diff
@@ -0,0 +1,80 @@
+import { analyticsStorage } from './storage'
```

Comment:
Let's make the numbers smaller 4 and 10 instead of 40 and 500.  Bigger numbers just use up more memory.